src/reducers/7.12/reducer.ts

```diff
@@ -884,7 +887,31 @@ const updateIngressDomain = (cr: BrokerCR, newName: string) => {
     if (!rt) {
       return;
     }
-    rt.patch.spec.tls[0].hosts = [certManagerTlsHost(cr, acceptor.name)];
+    rt.patch.spec.tls[0].hosts = certManagerTlsHosts(cr, acceptor.name);
+  });
+};
+
+const updateDeploymentSize = (cr: BrokerCR, newSize: number) => {
+  cr.spec.deploymentPlan.size = newSize;
+  if (cr.spec.deploymentPlan.size < 1) {
+    cr.spec.deploymentPlan.size = 1;
+  }
+  // when the size changes, some annotations will need an update to
+  // stay in sync
+  if (!cr.spec.acceptors) {
+    return;
+  }
+  if (!cr.spec.resourceTemplates) {
+    return;
+  }
+  cr.spec.acceptors.forEach((acceptor) => {
+    const rt = cr.spec.resourceTemplates.find(
```

Comment:
@lavocatt , Good work on this.
just a question here what is mean by variable `rt` here?

Reply:
`rt` is an abbreviation for `resource templace`.
`const resourceTemplate = cr.spec.resourceTemplates.find(` felt too verbose and redundant, since we're invoking `find` on the list `resourcesTemplates` we know we'll get a `resourceTemplate`, hence the abbreviation.

Reply:
got it thanks!